scripts/gha/ui_testing/uitest_apple/FirebaseCppUITestAppUITests/FirebaseCppUITestAppUITests.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  FirebaseCppUITestAppUITests.swift
+//  FirebaseCppUITestAppUITests
+//
+//  Created by Mou Sun on 3/19/22.
+//  Copyright © 2022 Google. All rights reserved.
+//
+
+import XCTest
+
+class FirebaseCppUITestAppUITests: XCTestCase {
+
+  override func setUp() {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+
+    // In UI tests it is usually best to stop immediately when a failure occurs.
+    continueAfterFailure = false
+
+    // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+    XCUIApplication().launch()
+
+    // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+  }
+
+  override func tearDown() {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+  }
+
+
+  func testGMA() {
+    let app = XCUIApplication(bundleIdentifier: "com.google.ios.admob.testapp")
+    app.launch()
+
+    // TestAdViewAdClick
+    var reference = app.staticTexts["Test mode"]
+    XCTAssertTrue(reference.waitForExistence(timeout: 60))
+    // click on the center point of the "Test Ad" TextView, where the Ad present
+    var x = (reference.frame.origin.x + reference.frame.width)/2
+    var y = (reference.frame.origin.y + reference.frame.height)/2
+    sleep(5)  // wait until button hittable
```

Comment:
nit: "wait until the button is clickable"

Reply:
Apple Developer Document uses hittable
https://developer.apple.com/documentation/xctest/xcuielement/1500561-hittable?language=objc